### ISSUE TITLE
Tag Full Site Editing is missing

### ISSUE THREAD

**Issue by kafleg** [WordPress/create-block-theme#313]
While checking Appearance > Create Block Theme, there is no option to select Full Site Editing tag. Without that tag, themes trac will not be recognized as Full Site Editing(Block Theme). 

I checked the function get_theme_feature_list() and found,  **Replaced 'Full Site Editing' feature name with 'Site Editor'.** 

But, I guess those tags should be aligned with the list of tags that we have in the Theme Check plugin and tags in the themes repository. 

Also, we don't allow custom tags in the theme for the WordPress themes repository, I don't think it is necessary to give the option to add custom tags. 

P.S. I assume that the generated themes will be submitted in the org themes repository.

**Comment by jffng** [comment]
> I checked the function get_theme_feature_list() and found, Replaced 'Full Site Editing' feature name with 'Site Editor'.

The theme is exported with the `full-site-editing` tag when `Site Editor` is checked. This seems like something to correct in the `get_theme_feature_list` function, if we wanted those to be consistent?

> Also, we don't allow custom tags in the theme for the WordPress themes repository, I don't think it is necessary to give the option to add custom tags.

Hmm, I think there may be use cases for adding a custom tag here. I think it's okay to leave the custom tags field in, but I don't have a strong opinion. Maybe we can add a note above the form field suggesting that custom tags are not allowed in the .org repo.

**Comment by carolinan** [comment]
> P.S. I assume that the generated themes will be submitted in the org themes repository.

I don't think that is the case for the majority of the themes. People just want to make changes and have their own theme that way.

**Comment by kafleg** [comment]
As a themes team rep, I think it is best to assume. haha
